### PR TITLE
Ensure executable bit on script directories

### LINKED ISSUE #89: missing executable bit in /start-entrypoint.d/001_set_report_url
I noticed that in camptocamp/odoo-project:11.0-latest the /start-entrypoint.d/001_set_report_url is missing the executable bit so it won't be executed.

    docker run --rm --entrypoint "/bin/bash" -ti  camptocamp/odoo-project:11.0-latest
    root@85c327e2a9ee:/# ls -l /start-entrypoint.d/
    total 8
    -rwxrwxr-x 1 root root 991 Oct 31 16:22 000_set_base_url
    -rw-rw-r-- 1 root root 648 Oct 31 16:22 001_set_report_url

### PR DESCRIPTION
**Pull request by pniederlag**
Fixes #89

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -30,7 +30,8 @@ on_exit() {
 }
 
 trap on_exit EXIT
-
+# as executable bit will be inherited to the docker image we explicitly set executable bit on scripts
+chmod start-entrypoint.d/* before-migrate-entrypoint.de/*
```
Comment (guewen) [inline build.sh]:
```suggestion
chmod start-entrypoint.d/* before-migrate-entrypoint.d/*
```

File: build.sh [hunk 2]
```diff
@@ -30,7 +30,8 @@ on_exit() {
 }
 
 trap on_exit EXIT
-
+# as executable bit will be inherited to the docker image we explicitly set executable bit on scripts
+chmod start-entrypoint.d/* before-migrate-entrypoint.d/*
```
Comment (guewen) [inline build.sh]: didn't see it before, the mode is missing :)

```suggestion
chmod +x start-entrypoint.d/* before-migrate-entrypoint.d/*
```

Comment (pniederlag) [reply build.sh]: Sry, fixed

**Comment by simahawk** [comment]
@pniederlag pls, rebase and check the issue pointed out by @yvaucher

**Pull request closed without merging**